proposals/lip-0050.md

```diff
@@ -64,6 +64,7 @@ We define the following constants:
 | Name | Type | Validation | Description |
 |------|------|------------|-------------|
 | `PublicKeyEd25519` | bytes | Must be of length 32. | Used for Ed25519 public keys. |
+| `Transaction` | object | Must follow the `transactionSchema` schema defined in [LIP 0068](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0068.md) with the only difference that `params` property is not serialized and contains the values of parameters of `paramsSchema` for the corresponding transaction. | An object representing a non-serialized transaction. |
```

Comment:
As mentioned, I would not define this type here, also because I think we want to move to a definition where `params` is still of type bytes

Reply:
Not a strong opinion on this on. Maybe better to remove it for now. We will not forget about this issue since it is already covered somewhere else.